src/VisualStudio/Core/Def/TableDataSource/Suppression/VisualStudioSuppressionFixService.cs

```diff
@@ -98,6 +99,7 @@ public async Task InitializeAsync(IAsyncServiceProvider serviceProvider)
 
         public bool AddSuppressions(IVsHierarchy? projectHierarchy)
         {
+            Logger.Log(FunctionId.Suppression_Add, logLevel: LogLevel.Information);
```

Comment:
i would prefer separate IDs for all of these.  so we know what entrypoint is being used.